dokka-runners/dokka-gradle-plugin/src/main/kotlin/internal/v2MigrationUtils.kt

```diff
@@ -47,14 +47,31 @@ private fun configureDokkaTaskConventions(project: Project) {
         outputDirectory.set(temporaryDir)
 
         doFirst("Disable Dokka V1 task") {
-            throw DokkaV1TaskDisabledException()
+            throw DokkaV1TaskDisabledException(
+                buildString {
+                    appendLine("Cannot run Dokka V1 tasks when V2 mode is enabled.")
+                    appendLine("Dokka Gradle plugin V1 mode is deprecated, and scheduled to be removed in Dokka v2.2.0.")
+                    appendLine("To finish migrating to V2 mode, please check the migration guide https://kotl.in/dokka-gradle-migration")
+                    append("Suggestion: ")
+                    when {
+                        "html" in this@task.name.lowercase() ->
+                            appendLine("Use `dokkaGenerate` or `dokkaGenerateHtml` tasks instead.")
+
+                        "javadoc" in this@task.name.lowercase() ->
+                            appendLine("Use `dokkaGenerate` or `dokkaGenerateJavadoc` tasks instead.")
+
+                        else ->
+                            appendLine("Use `dokkaGenerate` task instead.")
```

Comment:
Minor: There is probably no reason to suggest the `dokkaGenerate` task, as it will do nothing. I mean, we don't support other formats, so users will need to migrate from them :)